test/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ if( iodaconv_bufr_ENABLED )
     testinput/gdas.t00z.1bhrs4.tm00.bufr_d
     testinput/gdas.t06z.adpsfc.tm00.bufr_d
     testinput/gdas.t00z.sevcsr.tm00.bufr_d
+    testinput/NC005080.bufr_d
```

Comment:
@AshleyStanfield-NOAA , please reflect here (in CMakeLists.txt) the new file name(s), i.e.
testinput/NC005080.bufr_d will become testinput/bufr_ncep_satwind_avhrr.bufr_d, etc.

Reply:
I've updated CMakeLists to match the above. Will be re-uploaded shortly.